tests/benchmark_spec_test.py

```diff
@@ -97,16 +104,32 @@
 NEVER_SUPPORTED = 'mysql_service'
 NO_SUPPORT_INFO = 'this_is_not_a_benchmark'
 
+_DESCRIPTION = 'Test description.'
 
-class ConstructVmsTestCase(unittest.TestCase):
+
+class _BenchmarkSpecTestCase(unittest.TestCase):
 
   def setUp(self):
-    # Reset the current benchmark spec.
+    super(_BenchmarkSpecTestCase, self).setUp()
```

Comment:
I don't think this is necessary, since this class inherits directly from `unittest.TestCase`.